src/components/Searchbar/Searchbar.tsx

```diff
@@ -1,31 +1,25 @@
-import { CircleX, Search } from 'lucide-react';
+import { Input } from '@/components';
```

Comment:
Why don't to place the `x` button symmetric - with the same distance to border as the "🔍"

![image](https://github.com/user-attachments/assets/d5896bed-7e55-42a0-bc00-c9739efb21c8)

Reply:
Removed additional spacing